parquet-common/src/main/java/org/apache/parquet/bytes/MultiBufferInputStream.java

```diff
@@ -379,4 +427,120 @@ public void remove() {
       second.remove();
     }
   }
+
+  @Override
+  public byte readByte() throws IOException {
+    return (byte)readUnsignedByte();
+  }
+
+  @Override
+  public int readUnsignedByte() throws IOException {
```

Comment:
this looks the same as `read`

Reply:
You're right. They're identical. I think the C1 compiler will eliminate the overhead of read() calling readUnsignedByte().

Reply:
If they are identical, we should remove the duplicate one.

Reply:
Looks like I took care of this already.